src/main/java/org/verdictdb/coordinator/SelectQueryCoordinator.java

```diff
@@ -108,7 +119,8 @@ private SelectQuery standardizeQuery(String query) throws VerdictDBException {
     NonValidatingSQLParser sqlToRelation = new NonValidatingSQLParser();
     SelectQuery relation = (SelectQuery) sqlToRelation.toRelation(query);
     MetaDataProvider metaData = createMetaDataFor(relation);
-    RelationStandardizer gen = new RelationStandardizer(metaData);
+    ScrambleMetaStore metaStore = new ScrambleMetaStore(conn);
+    RelationStandardizer gen = new RelationStandardizer(metaData, metaStore);
```

Comment:
I think despite some redundancy of the code, it would be much cleaner to separate `RelationStandardizer` and the scramble table replacement logic.

Reply:
I will create a separate class for a substitute logic and let you know